ReleaseProcessAutomation/Steps/PipelineSteps/ReleaseAlphaBetaStep.cs

```diff
@@ -94,7 +95,34 @@ public void Execute (SemanticVersion nextVersion, string? commitHash, bool pause
 
     var nextJiraVersion = InputReader.ReadVersionChoice("Please choose next version (open JIRA issues get moved there):", nextPossibleJiraVersions);
 
-    _ijIraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+    _jiraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+
+    var currentFullVersion = nextVersion.GetCurrentFullVersion();
```

Comment:
Maybe we should move all of this logic into its own step. This would promote reuse, while also simplifying our logic inside as we can use `return` statements instead of all the nesting levels. What do you think?

Reply:
I agree that it should probably not be included inside the main part of the step, I would however not create a Jira Release Step just for this.

 I think a better option would be to just extract it into an extra method, either in the ReleaseAlphaBeta Step itself or the ReleaseProcessStepBase, as this would also promote reuse.
I dont think it would be beneficial to create a new step layer just for this one occurrence, as this would lead to like 5 new steps or some funky workaround just to get to the proper last step.

Reply:
I am fine with extracting this into a (base) method if we don't expect this functionality to be needed elsewhere. 

> as this would lead to like 5 new steps or some funky workaround just to get to the proper last step.

What do you mean by that? We would just move the code in the step and pass the step in the constructor. The benefit compared to the (base) method is that is that it is easier to reuse in another step. It is also easier to test as the logic can be tested in its isolated environment and the consuming steps can mock the inner step.

Reply:
"Funky workaround" was maybe a bit wrong yeah. We would still need to create lots of extra things though:

As we do not want steps to go back on themselves, we would need to create a following step  just for the other things happening in all the "Release..." steps like the specific ms build settings as well as up to 3 possible following steps in one release part. 

This might be a thing for an extra ticket, as there is some redundance overall in the steps, there is some improvement potential there.

Reply:
I will fix this by creating a base method for now, and will create a Task for a follow up ticket if that is ok for you.

Reply:
As moving all of jira things just came up in another pr i just did it and moved everything in that branch ([RMSRCBUILD-43](https://github.com/re-motion/SharedSource-Build/pull/89))

Reply:
We should probably finish 43 before we continue with this one as that should make this much more easy